Documentation/CRDs/Cluster/ceph-cluster-crd.md

```diff
@@ -937,3 +938,28 @@ However, all new configuration by the operator will be blocked with this cleanup
 Rook waits for the deletion of PVs provisioned using the cephCluster before proceeding to delete the
 cephCluster. To force deletion of the cephCluster without waiting for the PVs to be deleted, you can
 set the `allowUninstallWithVolumes` to true under `spec.CleanupPolicy`.
+
+## Ceph Config
+
+!!! warning
```

Comment:
How about moving the warning later in the section, such as after the the yaml example? It's not a critical warning, so mentioning the warning after describing the feature is better IMO.